sqlalchemy_trino/compiler.py

```diff
@@ -80,7 +80,13 @@
 
 
 class TrinoSQLCompiler(compiler.SQLCompiler):
-    pass
+    def limit_clause(self, select, **kw):
```

Comment:
It's better to add some docs why this method is needed.
You can see an example from `dialects/mysql/base.py`
![image](https://user-images.githubusercontent.com/6848311/143414403-d8711f3e-b78c-4fe7-93f4-04588e91c367.png)